### PR TITLE
Admin page displaying projects correctly

### PR DESCRIPTION
**Pull request by snack-attack**

### CODE REVIEW

File: admin.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?php
+require_once './database.php';
+
+//get projects
+$sql = "SELECT * FROM `project`;";
```
Comment (mporam) [inline admin.php:5]: NO SELECT *

File: admin.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+$query = $db->prepare($sql);
```
Comment (mporam) [inline admin.php:6]: dont need to prepare and execute if you have no user data, can just use query

File: admin.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            <caption>Projects</caption>
+            <thead>
+                <tr>
+                    <th scope="col">Project Title</th>
+                    <th scope="col">Actions</th>
+                </tr>
+            </thead>
+            <tbody>
+                <?php
+                    foreach($projects as $project) {
```
Comment (mporam) [inline admin.php:39]: what if projects is empty?

File: admin.php [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<?php
+require_once './database.php';
+
+//get projects
+$sql = "SELECT * FROM `project`;";
+$query = $db->prepare($sql);
+$query->execute();
+$projects = $query->fetchAll();
```
Comment (mporam) [inline admin.php:8]: should turn this into a function like `getAllProjects`, you could then reuse it on the front-end

**Review by mporam — CHANGES_REQUESTED** [review]

File: admin.php (hunk 3 above)
Comment (snack-attack) [reply admin.php:39]: added an if stment to check if $projects array isn't empty before running the loop

File: admin.php (hunk 2 above)
Comment (snack-attack) [reply admin.php:6]: will refactor this post-MVP

File: admin.php (hunk 4 above)
Comment (snack-attack) [reply admin.php:8]: will refactor this post-MVP

**Pull request merged**